restcomm/restcomm.dao/src/main/java/org/mobicents/servlet/restcomm/entities/Geolocation.java

```diff
@@ -0,0 +1,676 @@
+            this.dateUpdated = dateUpdated;
+        }
+
+        public void setAccountSid(Sid accountSid) {
+            this.accountSid = accountSid;
+        }
+
+        public void setSource(String source) {
+            this.source = source;
+        }
+
+        public void setDeviceIdentifier(String deviceIdentifier) {
+            this.deviceIdentifier = deviceIdentifier;
+        }
+
+        public void setGeolocationType(GeolocationType geolocationType) {
+            this.geolocationType = geolocationType;
+        }
+
+        public void setResponseStatus(String responseStatus) {
+            this.responseStatus = responseStatus;
+        }
+
+        public void setCellId(String cellId) {
+            this.cellId = cellId;
+        }
+
+        public void setLocationAreaCode(String locationAreaCode) {
+            this.locationAreaCode = locationAreaCode;
+        }
+
+        public void setMobileCountryCode(Integer mobileCountryCode) {
+            this.mobileCountryCode = mobileCountryCode;
+        }
+
+        public void setMobileNetworkCode(String mobileNetworkCode) {
+            this.mobileNetworkCode = mobileNetworkCode;
+        }
+
+        public void setNetworkEntityAddress(Long networkEntityAddress) {
+            this.networkEntityAddress = networkEntityAddress;
+        }
+
+        public void setAgeOfLocationInfo(Integer ageOfLocationInfo) {
+            this.ageOfLocationInfo = ageOfLocationInfo;
+        }
+
+        public void setDeviceLatitude(String devLatitude) {
+            this.deviceLatitude = devLatitude;
+        }
+
+        public void setDeviceLongitude(String devLongitude) {
+            this.deviceLongitude = devLongitude;
+        }
+
+        public void setAccuracy(Long accuracy) {
+            this.accuracy = accuracy;
+        }
+
+        public void setPhysicalAddress(String physicalAddress) {
+            this.physicalAddress = physicalAddress;
+        }
+
+        public void setInternetAddress(String internetAddress) {
+            this.internetAddress = internetAddress;
+        }
+
+        public void setFormattedAddress(String formattedAddress) {
+            this.formattedAddress = formattedAddress;
+        }
+
+        public void setLocationTimestamp(DateTime locationTimestamp) {
+            try {
+                this.locationTimestamp = locationTimestamp;
+            } catch (Exception exception) {
+                DateTime locTimestamp = DateTime.parse("1900-01-01");
+                this.locationTimestamp = locTimestamp;
+            }
+        }
+
+        public void setEventGeofenceLatitude(String eventGeofenceLat) {
+            this.eventGeofenceLatitude = eventGeofenceLat;
+        }
+
+        public void setEventGeofenceLongitude(String eventGeofenceLong) {
+            this.eventGeofenceLongitude = eventGeofenceLong;
+        }
+
+        public void setRadius(Long radius) {
+            if (geolocationType.toString().equalsIgnoreCase("Notification")) {
```

Comment:
better to use enum comparison if availabe rather than string comparison...
Also im not sure this business logic should be placed here, it seems a bit obscure